lib/swagger-builder.js

```diff
@@ -314,7 +314,7 @@ class SwaggerBuilder {
         type: 'object',
```

Comment:
@zoharbabin  any way you can see here that version property is relevant ONLY for ott target

Reply:
thanks for confirming. merged.